Automattic/wp-calypso
# Plans 2023: npm migration - useIsLargeCurrency updates

Related to https://github.com/Automattic/wp-calypso/pull/76442

## Proposed Changes

Tackles one of the TODOs in https://github.com/Automattic/wp-calypso/pull/76442. Moves `useIsLargeCurrency` under the grid folder and passes `siteId` to it.

## Testing Instructions



- Go to /plans/[free | paid] and /start/plans and ensure the grids render fine with correct prices
- Switch to INR locale and ensure plan prices do not overlap the column borders

## Pre-merge Checklist
